src/process/parent.rs

```diff
@@ -147,6 +147,7 @@ impl ChildChannel {
     pub fn wait_for_child_ready(&mut self, child_pid: Pid) -> Result<()> {
         // Create collection with capacity to store up to MAX_EVENTS events
         let mut events = Events::with_capacity(MAX_EVENTS);
+        log::error!("wait for child start");
```

Comment:
Should this be an error?

Reply:
Oops, I forgot to clear this log while debugging. I fixed it.